packages/config/lib/options.ts

```diff
@@ -6,13 +6,15 @@ interface ResolvedConfigOption {
   validation: Function
   isFolder?: boolean
   isExperimental?: boolean
+  isWriteable?: boolean
 }
 
 interface RuntimeConfigOption {
   name: string
   defaultValue: any
   validation: Function
   isInternal?: boolean
+  isWriteable?: boolean
```

Comment:
Should any run-time configuration options be writeable?

Reply:
I believe there are some that aren't writeable because they either might be internal and we wouldn't want users to have the ability to change them or it's too late to change them to influence anything

Reply:
I guess my question here is shouldn't all run-time configuration have isWriteable = false?

Reply:
that's correct for these configuration options- those should be updated now